stores/identity.ts

```diff
@@ -153,7 +154,6 @@ export const useIdentityStore = defineStore('identity', {
     },
     async setAuth(authRequest: Auth) {
       this.auth = { ...authRequest, balance: DEFAULT_BALANCE_STATE }
-      this.resetMultipleBalances()
```

Comment:
why remove?

Reply:
`setAuth` is also used when changing the prefix/chain. for multi balances, no need to reset that. because the address and the balance are already stored in multi chain